MvvmCross/Binding/Binders/MvxNamedInstanceRegistryFiller.cs

```diff
@@ -82,6 +82,8 @@ where type.IsConventional()
             {
                 try
                 {
+                    if (pair.Type.ContainsGenericParameters) continue;
```

Comment:
What does this do?

Reply:
It prevents exceptions being raised when the Type is a generic - In the playground forms droid sample it is throwing an exception.